packages/material-ui-lab/src/DateRangePicker/DateRangePicker.test.tsx

```diff
@@ -268,12 +268,12 @@ describe('<DateRangePicker />', () => {
         open
         renderInput={defaultRangeRenderInput}
         onChange={() => {}}
-        renderDay={(day) => <div key={String(day)} data-mui-test="renderDayCalled" />}
+        renderDay={(day) => <div key={String(day)} data-testid="renderDayCalled" />}
```

Comment:
Only change touching pickers since I didn't need to alter pickers implementation